x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/geo_view/settings.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { BucketSpan } from '../bucket_span';
+import { SplitFieldSelector } from '../split_field';
+import { Influencers } from '../influencers';
+
+interface Props {
+  setIsValid: (proceed: boolean) => void;
+}
+
+export const GeoSettings: FC<Props> = ({ setIsValid }) => {
+  return (
+    <>
+      <EuiFlexGroup gutterSize="xl">
+        <EuiFlexItem>
+          <SplitFieldSelector />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <Influencers />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiFlexGroup gutterSize="xl">
+        <EuiFlexItem>
+          <BucketSpan setIsValid={setIsValid} />
```

Comment:
Does the bucket span estimator currently work with lat_long jobs, and if not, is it possible to hide the 'Estimate bucket span' button from the wizard?

<img width="985" alt="image" src="https://user-images.githubusercontent.com/7405507/206257788-9bc28cba-2453-4e28-9de5-bc825818b11c.png">

Reply:
Leave this for a follow-up. Could you raise an issue for it please.